docs/install.rst

```diff
@@ -59,6 +59,9 @@ If you installed Python from source, with an installer from `python.org`_, or
 via `Homebrew`_ you should already have pip. If you're on Linux and installed
 using your OS package manager, you may have to `install pip <https://pip.pypa.io/en/stable/installing/>`_ separately.
 
+If you plan to install pipenv using Homebrew you can skip this step. The
```

Comment:
I don't know if this is necessary.

Reply:
I would like this to stay. System python on macOS doesn't include pip. Homebrew python doesn't either, it gives you either `pip2` or `pip3` but no `pip`.

So its very likely that `pip --version` will fail for anyone interested in the Homebrew install method. There's no point making them go and work out how to install pip for themselves when it's not needed.

I'm not actually sure if it's needed for the Pipsi or Nix method either.